packages/salesforcedx-vscode-apex-replay-debugger/package.json

```diff
@@ -106,6 +106,10 @@
         {
           "command": "sfdx.launch.replay.debugger.logfile",
           "when": "sfdx:project_opened && editorLangId == 'apexlog'"
+        },
+        {
+          "command": "sfdx.launch.replay.debugger.last.logfile",
+          "when": "sfdx:project_opened && !inDebugMode"
```

Comment:
This command is available when you're in an SFDX project and not already debugging something. If you're already debugging something the Restart Debugging commands effectively do the same thing, restart your debugging session with the same parameters.